### PR TITLE
MB-10084: Add storage facility info/address to ServicesCounselingShipmentForm

### PR DESCRIPTION
**Pull request by kylehilltruss**
## Description

This pull request creates new components to collect the storage facility info and address, and adds those new components as sections to the `ServicesCounselingShipmentForm` for NTS-release shipments only.

## Reviewer Notes

Ensure that everything looks correct in both phone and desktop view. Ensure the form is behaving correctly -- especially when it comes to validation.

## Setup

```sh
yarn storybook
```

## Code Review Verification Steps

* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* Closes [MB-10084](https://dp3.atlassian.net/browse/MB-10084).

## Screenshots

* [Form with new components, at phone width](https://user-images.githubusercontent.com/83614364/144135006-55bdae33-6d0b-46e6-8de7-ef0f0c2b0240.png)
* [Form with new components, at desktop width](https://user-images.githubusercontent.com/83614364/144135146-c91be80b-d94c-4843-b3f9-2131167dc722.png)

### CODE REVIEW

File: src/components/Office/ShipmentWeightInput/ShipmentWeightInput.module.scss [hunk 1]
```diff
@@ -1,3 +1,12 @@
...
+
+.Fieldset {
+  h2 {
+    @include u-margin-bottom(2);
+  }
+}
+
 .Label {
   span {
     font-weight: normal;
```
Comment (YanZ777) [inline src/components/Office/ShipmentWeightInput/ShipmentWeightInput.module.scss:12]: I'm a bit leery of the duplicated css I see across these files. Is there a way to combine them?

**Review by amanda5iu — APPROVED** [review]
LGTM!

File: src/components/Office/ShipmentWeightInput/ShipmentWeightInput.module.scss (hunk 1 above)
Comment (kylehilltruss) [reply src/components/Office/ShipmentWeightInput/ShipmentWeightInput.module.scss:12]: Composited some css files to make things more DRY.

**Review by YanZ777 — APPROVED** [review]
Looks good to me. Thanks for combining the css.

**Pull request merged**